README.md

```diff
@@ -27,7 +27,7 @@ If you are not using timezone but only a few simple functions from moment.js, th
 
 > &mdash;<cite>Jared Farago from [webnode](https://github.com/oysterprotocol/webnode/pull/116) project.</cite>
 
-> [Make use of native JavaScript object and array utilities before going big. Good library if you’re looking to replace Moment.js for one reason or another. Immutable too.](https://twitter.com/dan_abramov/status/805030922785525760)
```

Comment:
Where is this sentence from @ddvkid @cht8687 ?

Reply:
It sounds like my word 😆

Reply:
Anyway I don't mind merging it since I can't recall where was it come from.

Reply:
@stevemao I probably made a mistake in this commit: 
https://github.com/you-dont-need/You-Dont-Need-Momentjs/commit/817b35803c87a12cfed5d1d6f4b01af3df40ffcb

The PR can be merged. But why the build is failed?